.azure-pipelines-templates/push_perf_data.yml

```diff
@@ -19,6 +19,13 @@ steps:
     workingDirectory: build
 
 # cimetrics related tasks: render and publish results for Pull Requests build
+- script: ./cimetrics_env.sh python upload_pico_metrics.py
```

Comment:
I think we always want to upload, it's just the plotting/posting we only want to do on PR.